docs/content/language-reference.md

```diff
@@ -159,7 +159,159 @@ The following table shows examples of how ``glob.match`` works:
 | <span class="opa-keep-it-together">``output := yaml.marshal(x)``</span> | ``output`` is ``x`` serialized to a YAML string |
 | <span class="opa-keep-it-together">``output := yaml.unmarshal(string)``</span> | ``output`` is ``string`` deserialized to a term from YAML encoded string |
 
-### Tokens
+### Token Signing
+
+OPA provides two builtins that implement JSON Web Signature [RFC7515](https://tools.ietf.org/html/rfc7515) functionality. 
+
+``io.jwt.encode_sign_raw()`` takes three JSON Objects (strings) as parameters and returns their JWS Compact Serialization. This builtin should be used by those that want maximum control over the signing and serialization procedure. It is important to remember that StringOrURI values are compared as case-sensitive strings with no transformations or canonicalizations applied. Therefore, line breaks and whitespaces are significant.
+
+``io.jwt.encode_sign()`` takes three Rego Objects as parameters and returns their JWS Compact Serialization. This builtin should be used by those that want to use rego objects for signing during policy evaluation. Notice that since these are objects canonicalization *is applied* and all whitespace is removed.
```

Comment:
Nit: this function takes JSON objects.

Reply:
I think your definition of JSON objects and the RFC are not in synch.

Reply:
The Rego language docs refer to the types you interact with inside the language not the bits on the wire. Please update the docs accordingly.

Reply:
Done